src/harness/testcases/WINNF_FT_S_MCP_testcase.py

```diff
@@ -46,6 +46,10 @@ def generate_MCP_1_default_config(self, filename):
       open(os.path.join('testcases', 'testdata', 'device_e.json')))
     device_6 = json.load(
       open(os.path.join('testcases', 'testdata', 'device_f.json')))
+    device_7 = json.load(
+      open(os.path.join('testcases', 'testdata', 'device_g.json')))
+    device_8 = json.load(
```

Comment:
This device is Cat B so you need to load conditional reg data.

Reply:
Updated   the conditional of device_8